CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# Release 0.14.0-dev
+
+### New features
+
+### Improvements
+
+### Bug fixes
+
+### Contributors
```

Comment:
Oh, should there be a documentation here as well?
```suggestion
### Bug fixes

### Documentation

### Contributors
```

Reply:
Oops, good catch!